caput/mpiarray.py

```diff
@@ -245,6 +236,71 @@ class MPIArray(np.ndarray):
         The dimension to distribute the array across.
     """
 
+    def __getitem__(self, v):
+        # Return an MPIArray view
+
+        if not isinstance(v, tuple):
+            return super().__getitem__(v)
```

Comment:
What happens for this?
```
dist_arr = mpiarray.MPIArray((4,), axis=0, comm=comm)
a = dist_arr[0]
```
I think this should fail because it indexes the distributed axis, but because of this if statement it would actually work because it just calls the numpy implementation.